tests/test_backend_loki_fields.py

```diff
@@ -0,0 +1,82 @@
+from typing import Any, Dict
+import pytest
+from sigma.backends.loki import LogQLBackend
+from sigma.collection import SigmaCollection
+from sigma.exceptions import SigmaFeatureNotSupportedByBackendError, SigmaTypeError
+from sigma.modifiers import modifier_mapping
+
+
+@pytest.fixture
+def loki_backend() -> LogQLBackend:
+    return LogQLBackend()
+
+
+# Mapping from modifier identifier strings to modifier classes
+modifier_sample_data: Dict[str, Any] = {
+    "contains": "valueA",
+    "startswith": "valueA",
+    "endswith": "valueA",
+    "exists": "yes",
+    "base64": "valueA",
+    "base64offset": "valueA",
+    "wide": "valueA",
+    "windash": "-foo",
+    "re": ".*valueA$",
+    "i": "valueA",
+    "ignorecase": "valueA",
+    "m": ["valueA", "valueB"],
+    "multiline": ["valueA", "valueB"],
+    "s": "valueA",
+    "dotall": "valueA",
+    "cased": "valueA",
+    "cidr": "192.0.0.0/8",
+    "all": ["valueA", "valueB"],
+    "lt": 1,
+    "lte": 1,
+    "gt": 1,
+    "gte": 1,
+    "fieldref": "fieldA",
+    "expand": '"%test%"',
+}
+
+
+def generate_rule_with_field_modifier(modifier: str, value: str) -> str:
```

Comment:
And then here, change this to return a tuple

```suggestion
def generate_rule_with_field_modifier(modifier: str, value: str) -> tuple[str, str]:
```

Reply:
Addressed.